volume-group-operator/controllers/utils/utils.go

```diff
@@ -0,0 +1,31 @@
+/*
+Copyright 2022.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package utils
+
+import (
+	corev1 "k8s.io/api/core/v1"
+)
+
+func removeByIndexFromPersistentVolumeClaimList(pvcList []corev1.PersistentVolumeClaim,
```

Comment:
sometimes the name with PVC, and others with PersistentVolumeClaim, can we unify to PVC? (same for PV)
```suggestion
func removeByIndexFromPVCList(pvcList []corev1.PersistentVolumeClaim,
```

Reply:
done

Reply:
and created CSI-5156 ticket